docs/idr-s3.md

```diff
@@ -0,0 +1,21 @@
+# IDR S3 upload server
+
+The IDR S3 submission server is co-located with the IDR FTP server.
```

Comment:
This made me wonder: we have no similar documentation describing the FTP server deployment?